src/buffers/gfx_output_buffer.c

```diff
@@ -1,4 +1,4 @@
 #include <ultra64.h>
 #include "gfx_output_buffer.h"
 
-u64 gGfxSPTaskOutputBuffer[0x2fc0];
+u64 gGfxSPTaskOutputBuffer[0x2000];
```

Comment:
This is like ~30kb of memory reduction; was the extra data really actually completely useless? Or does it come with any associated risks to be aware of?

Reply:
RSP slows down a bit. There is so much RSP overhead that it has no real world effect.

Reply:
How does know when to stop accumulating data? Is it just a `sizeof` somewhere?

Reply:
Yeah, SPTask gets two addresses, the start, and the end.